src/Directory.Build.targets

```diff
@@ -12,6 +12,11 @@
 
   <Import Project="..\Directory.Build.targets" />
 
+  <PropertyGroup Condition="'$(IsInvokableTestProject)' == 'true'">
```

Comment:
I would put this directly above the target that we are using it with so they are close together.